core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -537,7 +537,11 @@ private static TabletMetadata getRootMetadata(ClientContext ctx,
       case IMMEDIATE:
         ZooReader zooReader = ctx.getZooReader();
         try {
-          byte[] bytes = zooReader.getData(zkRoot + RootTable.ZROOT_TABLET);
+          var path = zkRoot + RootTable.ZROOT_TABLET;
```

Comment:
I wonder if eventual should even be supported for the root table. Maybe it should always be immediate regardless of consistency level.

Reply:
I think the evenutal  case was for code that used to use zoocache to read root tablet metadata that was adpated to use Ample.  That code did not care if it got the latest and greatest, it just wanted performance.